.travis/setenv.sh

```diff
@@ -0,0 +1,9 @@
+if [[ "${build_with_stack}" ]]; then
+  mkdir -p ~/.local/bin
+  export PATH=$HOME/.local/bin:$PATH
+  travis_retry curl -L https://get.haskellstack.org/stable/linux-x86_64.tar.gz | tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack'
+else
+  export PATH=$HOME/.cabal/bin:$PATH
+  nvm install node
+  nvm use node
```

Comment:
Is there a reason, why you only install `node` when building with cabal?

Reply:
That's mistake. I'll fix it.